pages/configure_links.php

```diff
@@ -74,11 +74,11 @@
 	if ( $t_custom_field_def['type'] == CUSTOM_FIELD_TYPE_ENUM ||
 		 $t_custom_field_def['type'] == CUSTOM_FIELD_TYPE_MULTILIST)
 	{
-		echo '<td>'. string_display_line( $t_custom_field_def['name'] ).'</td>';
+		echo '<td>'. string_attribute( $t_custom_field_def['name'] ).'</td>';
```

Comment:
Technically this change is not necessary, string_display_line() will prevent XSS as well.